### PR TITLE
ceph: pickup 12.2.1

### PR DESCRIPTION
**Pull request by bassam**

### CODE REVIEW

File: images/ceph/commit_hash [hunk 1]
```diff
@@ -1 +1 @@
-cec053b62872f1342c8ebd8bc6d5e9c21aa82a1e
\ No newline at end of file
+3e7492b9ada8bdc9a5cd0feafd42fbca27f9c38e
```
Comment (jbw976) [inline images/ceph/commit_hash]: in the past you've rebased your change to "remove boost submodule" on top of the latest commits from upstream ceph.  is that no longer needed?

**Comment by jbw976** [comment]
Any breaking changes that you are aware of or that we need to pay special attention to?

File: images/ceph/commit_hash (hunk 1 above)
Comment (bassam) [reply images/ceph/commit_hash]: ahh I missed that. fixed. its not strictly needed but does improve the build speed.

**Comment by bassam** [comment]
no breaking changes that I know of.

**Pull request merged**